spec/factories/form526_job_statuses.rb

```diff
@@ -30,10 +33,44 @@
     error_message { 'undefined method foo for nil class' }
     bgjob_errors do
       {
-        error_class: 'NoMethodError',
-        error_message: 'undefined method foo for nil class',
-        timestamp: Time.now.utc
+        Time.now.utc.to_i.to_s => {
+          'caller_method' => 'non_retryable_error_handler',
+          'error_class' => 'NoMethodError',
+          'error_message' => 'undefined method foo for nil class',
+          'timestamp' => Time.now.utc
+        }
       }
     end
   end
+
+  # rubocop:disable Layout/LineLength
+  trait :pif_in_use_error do
+    job_class { 'SubmitForm526AllClaim' }
+    status { 'exhausted' }
+    error_class { 'EVSS::DisabilityCompensationForm::ServiceException' }
+    error_message { 'PIF in use' }
+    bgjob_errors do
+      {
+        Time.now.utc.to_i.to_s => {
+          'caller_method' => 'job_try',
+          'timestamp' => '2021-11-09T15:55:58.639Z',
+          'error_class' => nil,
+          'error_message' => nil
+        },
+        Time.now.utc.to_i.to_s => {
+          'caller_method' => 'job_exhausted',
+          'timestamp' => '2021-11-09T15:56:05.708Z',
+          'error_class' => 'EVSS::DisabilityCompensationForm::ServiceException',
+          'error_message' => 'PIF in use'
+        },
+        Time.now.utc.to_i.to_s => {
+          'caller_method' => 'retryable_error_handler',
+          'timestamp' => '2021-11-09T15:56:05.702Z',
+          'error_class' => {},
+          'error_message' => '[{"key"=>"form526.submit.save.draftForm.PIFInUse", "severity"=>"FATAL", "text"=>"Claim could not be established. Contact the BDN team and have them run the WIPP process to delete Cancelled/Cleared PIFs"}]'
```

Comment:
```suggestion
          'error_message' => '[{"key"=>"form526.submit.save.draftForm.PIFInUse", "severity"=>"FATAL", ' \
            '"text"=>"Claim could not be established. Contact the BDN team and have them run the WIPP ' \
            'process to delete Cancelled/Cleared PIFs"}]'
```

Reply:
I'd recommend splitting up the line so you don't have to disable the line length cop.